README.rst

```diff
@@ -90,8 +90,10 @@ Here is an example on how to use the ``GLM`` estimator.
     print('Pseudo R^2 is %.3f' % pseudo_R2)
 
     # plot the true coefficients and the estimated ones
-    plt.stem(beta, markerfmt='r.', label='True coefficients')
-    plt.stem(glm.beta_, markerfmt='b.', label='Estimated coefficients')
+    plt.stem(beta, markerfmt='r.', label='True coefficients',
+             use_line_collection=True)
```

Comment:
don't use this ... it will fail with matplotlib 3.0 and matplotlib 3.1 does not support python 3.5.